src/style.css

```diff
@@ -0,0 +1,21 @@
+body {
+  margin: 0;
+}
+
+.block1,
+.block2,
+.block3 {
+  height: 100px;
+}
```

Comment:
good idea to write this separately and one more thing to improve: add a general class for these elements and add it to corresponding tags in HTML file